test/runtests.jl

```diff
@@ -63,6 +63,8 @@ forcing_sbm_gw_path = testdata(
 forcing_meuse_path = testdata(v"0.2.8", "forcing_meuse.nc", "forcing_meuse.nc")
 staticmaps_sbm_gw_path =
     testdata(v"0.2.2", "staticmaps-sbm-groundwater.nc", "staticmaps-sbm-groundwater.nc")
+instates_sbm_gw_path =
+    testdata(v"0.2.2", "instates-example-sbm-gwf.nc", "instates-example-sbm-gwf.nc")
```

Comment:
I think for now this is fine, as this data is only used in the current tests (as the `reinit` flag gets set in memory rather than in the .toml). But just to be aware that we also have a download_test_data.jl script in the build/create_binaries folder, with some duplicate code (as mentioned in #365). No something to pick up in the PR I'd say, but something to fix in the (near) future.

Reply:
See commit https://github.com/Deltares/Wflow.jl/pull/371/commits/59f54b41a32d3450a88f7c22bb7a0af945f7a7f6.